modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetRequest.java

```diff
@@ -119,18 +121,18 @@ public void start(NHttpClientConnection conn) throws IOException, HttpException
             TargetContext.get(conn).setWriter(pipe);
         }
 
-        String path = fullUrl || (route.getProxyHost() != null && !route.isTunnelled()) ?
-                    url.toString() : url.getPath() +
-                    (url.getQuery() != null ? "?" + url.getQuery() : "");
+        String path = fullUrl || (route.getProxyHost() != null && !route.isTunnelled()) ? url.toString() : url.getPath() + (url.getQuery() != null ? "?" + url.getQuery() : "");
```

Comment:
This line seems to be exceeding 120 limit.

Reply:
I changed the code that you mentioned.